cmd/swag/main.go

```diff
@@ -67,13 +68,18 @@ var initFlags = []cli.Flag{
 		Usage:   "Parse folder containing markdown files to use as description, disabled by default",
 	},
 	&cli.BoolFlag{
-		Name:  "parseInternal",
+		Name:  parseInternalFlag,
 		Usage: "Parse go files in internal packages, disabled by default",
 	},
 	&cli.BoolFlag{
-		Name:  "generatedTime",
+		Name:  generatedTimeFlag,
 		Usage: "Generate timestamp at the top of docs.go, true by default",
 	},
+	&cli.IntFlag{
+		Name:  parseDepthFlag,
```

Comment:
@necryin could you help pull a new PR to update  https://github.com/swaggo/swag#swag-cli as well ?